lib/RoleDetails/utils.js

```diff
@@ -100,3 +100,35 @@ export async function createUserRolesRequests(previousSelectedUsers, currentSele
 
   return requests;
 }
+
+/**
+ * Asynchronously checks the authentication of Keycloak users based on the provided user IDs.
+ *
+ * @param {Object} options - The options object.
+ * @param {Array} options.userIds - An array of user IDs to check authentication for.
+ * @param {Object} options.queryClient - The query client object for fetching data.
+ * @param {Object} options.ky - The ky instance for making HTTP requests.
+ */
+
+export async function checkAndGetNoneKeycloakAuthUsers({ roleId, userIds, queryClient, ky }) {
+  const keycloakNoneExistingUsers = [];
```

Comment:
Again here, I'd prefer a simple, but clear variable name:

```suggestion
  const nonKeycloakUsers = [];
```